src/calculateRentalCost.js

```diff
@@ -28,6 +28,17 @@
  */
 
 function calculateRentalCost(days) {
+  const fullCost = days * 40;
```

Comment:
Please pay attention on [checklist](https://github.com/oolcha/-js_task-transportation-on-vacation/blob/main/checklist.md)
Make variables for `magic numbers`

Reply:
Done